shared-libs/transitions/src/transitions/muting.js

```diff
@@ -40,14 +40,23 @@ const isRelevantReport = (doc, info = {}) =>
           !transitionUtils.hasRun(info, TRANSITION_NAME) &&
           utils.isValidSubmission(doc));
 
-// when new contacts are added that have muted parents, they should be set have muted state as well
-// also the schedule associated with their registration should be muted
-const isRelevantContact = (doc, info = {}) =>
+//
+// When *new* contacts are added that have muted parents, they and their schedules should be muted.
+//
+// We are deciding a contact is new if:
+//  - They were initially replicated *after* a mute that has happened in their parent lineage
+//  - And we haven't performed any kind of mute on them before
+//
+const isRelevantContact = (doc, infoDoc = {}) =>
   Boolean(doc &&
-          !info._rev &&
+          ['person', 'clinic', 'health_center', 'district_hospital'].includes(doc.type) &&
```

Comment:
I think this has to be removed, I think it's old code that does not work with configurable hierarchy, also the configurable  hierarchy check is on next line (`isContact(doc)`): https://github.com/medic/medic/blob/master/shared-libs/transitions/src/transitions/muting.js#L48